bosh-director/lib/bosh/director/jobs/update_release.rb

```diff
@@ -39,13 +41,15 @@ def perform
         logger.info("Processing update release")
         logger.info("Release rebase will be performed") if @rebase
 
-        single_step_stage("Downloading remote release") { download_remote_release } if @release_url
+        single_step_stage("Downloading remote release") { download_remote_release } if release_url
 
         release_dir = nil
         single_step_stage("Extracting release") { release_dir = extract_release }
 
         single_step_stage("Verifying manifest") { verify_manifest(release_dir) }
 
+        single_step_stage("Verifying remote release") { verify_sha1 } if release_url && sha1
```

Comment:
Let's also verify sha1 even if it's a locally uploaded tarball -> if sha1 (just like stemcells)